### DISCUSSION TITLE
Zustand/persist middleware with Typescript.

### DISCUSSION THREAD

**Discussion by PranuPranav97** [pmndrs/zustand#1350] [Q&A] [answered]
Hi, I want to use Zustand with typescript. I am not able to find an example for using zustand/persist middleware in typescript.

**Comment by dai-shi** [comment]
Doesn't this help? https://github.com/pmndrs/zustand/blob/a848f708b747bc6b3fd5eaba8e5589df9e997de9/docs/guides/typescript.md#using-middlewares

**Comment by PranuPranav97** [comment, marked as answer]
Yeah. I was not able to understand in the first place. Also finally made persist globally in nextjs [Check it Here](https://github.com/PranuPranav97/zustand-persist-global)
